.github/workflows/codecov.yml

```diff
@@ -0,0 +1,46 @@
+name: codecov
+on:
+  push:
+    branches: [main, 0.4.x]
+  pull_request:
+jobs:
+  # Run code coverage using cargo-llvm-cov then upload to codecov.io
+  job_code_coverage:
+    name: llvm-cov
+    runs-on: ubuntu-latest
+    env:
+      # `MSRV` should match project `cargo-llvm-cov` value for `rust-version`
+      MSRV: 1.64
```

Comment:
The oldest version supported with pre-build binaries is 1.60.
But is it not best to just use the latest rust version?

Reply:
Removed the `MSRV` setting